crates/re_space_view_spatial/src/eye.rs

```diff
@@ -454,7 +467,9 @@ impl OrbitEye {
         let rot_delta = Quat::from_rotation_z(delta_angle);
         self.world_from_view_rot *= rot_delta;
 
-        self.up = Vec3::ZERO; // forget about this until user resets the eye
+        // Permanently change our up-axis, at least until the user resets the view!
+        let up = self.view_coordinates.up().map_or(glam::Vec3::Y, Into::into);
+        self.up = self.world_from_view_rot.mul_vec3(up).normalize_or_zero();
```

Comment:
let's use different names for the different up-axes

```suggestion
        let scene_up = self.view_coordinates.up().map_or(glam::Vec3::Y, Into::into);
        self.up = self.world_from_view_rot.mul_vec3(scene_up).normalize_or_zero();
```